sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/CollectionCrudTest.java

```diff
@@ -367,6 +370,33 @@ public void sessionTokenConsistencyCollectionDeleteCreateSameName() {
         }
     }
 
+    @Test(groups = { "emulator" }, timeOut = TIMEOUT)
+    public void readReplaceAutoscaleThroughput() throws Exception {
+        safeDeleteDatabase(client.getDatabase("newTestDatabase"));
```

Comment:
tests should not use static database name to avoid any potential issue with runnings tests in parallel.

Please use CosmosDatabaseForTest.generateId()

Reply:
Done